client/extensions/woocommerce/app/dashboard/manage-orders-view.js

```diff
@@ -84,45 +83,60 @@ class ManageOrdersView extends Component {
 		}
 	};
 
+	shouldShowPendingReviews = () => {
+		return config.isEnabled( 'woocommerce/extension-reviews' ) && this.props.pendingReviews;
+	};
+
 	possiblyRenderProcessOrdersWidget = () => {
-		const { site, orders, ordersRevenue, currency } = this.props;
+		const { currency, orders, ordersRevenue, site, translate } = this.props;
 		if ( ! orders.length ) {
 			return null;
 		}
+		const currencyValue = ( currency && currency.value ) || '';
+		const orderCountPhrase = translate( '✨ New order', '✨ New orders', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'New Order' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-dashboard-widgets)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).